Python/codecs.c

```diff
@@ -50,6 +50,26 @@ int PyCodec_Register(PyObject *search_function)
     return -1;
 }
 
+int
+PyCodec_Unregister(PyObject *search_function)
+{
+    PyInterpreterState *interp = PyInterpreterState_Get();
+    /* Do nothing if codec_search_path is not created yet or was created. */
```

Comment:
Oops, this typo is likely mine, sorry :-)

```suggestion
    /* Do nothing if codec_search_path is not created yet or was cleared. */
```

Reply:
updated ;)